ui/src/App.js

```diff
@@ -47,6 +48,12 @@ class App extends Component {
       } else {
         this.setState({
           fields: data.fields.map(field => {
+            if (field.description == null || field.description === "") {
```

Comment:
![longchip](https://user-images.githubusercontent.com/10929390/45848033-de6aba00-bce1-11e8-9f92-17d7b1cf2261.png)

The chip can be long, see image above. We should show "field name - description" in the drop-down, but only field name in the chip. Can be a separate PR. I [asked about this on StackOverflow](https://stackoverflow.com/questions/52433569/react-select-show-different-text-label-for-drop-down-and-control).